.github/workflows/aks.yaml

```diff
@@ -20,7 +20,13 @@ env:
 
 jobs:
   installation-and-connectivity:
-    if: ${{ (github.event.issue.pull_request && startsWith(github.event.comment.body, 'ci-aks')) || github.event_name == 'push' || github.event.label.name == 'ci-run/aks' }}
+    if: |
+      (github.event.issue.pull_request && (
+        startsWith(github.event.comment.body, 'ci-aks') ||
+        startsWith(github.event.comment.body, 'test-me-please')
```

Comment:
@aanm Since you just happened to discover that connectivity tests are actually ran from GH Actions, which obviously locks a GH Actions slot as long as a test is running, can you double-confirm that we add the new CI 3.0 workflows to be triggered by `test-me-please`?

Reply:
You can have a look at some "sample length" from the workflows linked in comment. Do note that EKS does not encryption enabled yet.

Reply:
@nbusseneau Let's try it out and see how it goes.